index.html

```diff
@@ -280,21 +280,16 @@
       var content = encodeURIComponent(editor.getSession().getDocument().getValue());
 
       var xhr = new XMLHttpRequest();
-      xhr.open("POST", "https://api-ssl.bitly.com/v4/shorten", true);
+      xhr.open("POST", "http://localhost:8888/api/", true);
```

Comment:
Поскольку у нас подразумевается, что все под одним сервером, то можно просто вот так:
```suggestion
      xhr.open("POST", "/api/", true);
```